aspnetcore/fundamentals/app-state.md

```diff
@@ -16,28 +16,26 @@ ms.custom: H1Hack27Feb2017
 
 # Introduction to session and application state in ASP.NET Core
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Steve Smith](http://ardalis.com)
+By [Rick Anderson](https://twitter.com/RickAndMSFT), [Steve Smith](http://ardalis.com)  and [Diana LaRose](https://github.com/DianaLaRose)
```

Comment:
@DianaLaRose Hi -- Lists were following the Oxford/Harvard/Serial comma rule ... unless something has changed. Two spaces won't be rendered, but the practice has been to eliminate them when discovered. There's one double-space on this line and three below in the new text